### ISSUE TITLE
[kunlun ]  paddleclas 运行vgg16出错

### ISSUE THREAD

**Issue by 1chang37** [PaddlePaddle/PaddleClas#1785]
k200+kylinv10 运行出现如下错误

[root@localhost PaddleClas]# bash run_vgg16.sh 
WARNING: AVX is not support on your machine. Hence, no_avx core will be imported, It has much worse preformance than avx core.
[INFO][BKCL][/home/qa_work/workspace/xpu_build_dailyjob/bkcl_root/baidu/xpu/bkcl/src/globals.cpp:51] set BKCL timeout to 600 seconds
[INFO][BKCL][/home/qa_work/workspace/xpu_build_dailyjob/bkcl_root/baidu/xpu/bkcl/src/globals.cpp:52] set BKCL RING BUFFER SIZE to 1048576
XPURT /usr/local/lib64/python3.7/site-packages/paddle/fluid/../libs/libxpurt.so loaded
2022-03-24 16:45:28,670 - WARNING - A new filed (use_gpu) detected!
2022-03-24 16:45:28,670 - WARNING - A new filed (use_cpu) detected!
2022-03-24 16:45:28,671 - WARNING - A new filed (use_xpu) detected!
2022-03-24 16:45:28,671 - WARNING - A new filed (is_distributed) detected!
2022-03-24 16:45:28,671 - INFO - 
===========================================================
==        PaddleClas is powered by PaddlePaddle !        ==
===========================================================
==                                                       ==
==   For more info please go to the following website.   ==
==                                                       ==
==       https://github.com/PaddlePaddle/PaddleClas      ==
===========================================================

2022-03-24 16:45:28,671 - INFO - ARCHITECTURE : 
2022-03-24 16:45:28,671 - INFO -     name : VGG16
2022-03-24 16:45:28,671 - INFO - ------------------------------------------------------------
2022-03-24 16:45:28,671 - INFO - LEARNING_RATE : 
2022-03-24 16:45:28,672 - INFO -     function : Cosine
2022-03-24 16:45:28,672 - INFO -     params : 
2022-03-24 16:45:28,672 - INFO -         lr : 0.0005
2022-03-24 16:45:28,672 - INFO - ------------------------------------------------------------
2022-03-24 16:45:28,672 - INFO - OPTIMIZER : 
2022-03-24 16:45:28,672 - INFO -     function : Momentum
2022-03-24 16:45:28,672 - INFO -     params : 
2022-03-24 16:45:28,673 - INFO -         momentum : 0.9
2022-03-24 16:45:28,673 - INFO -     regularizer : 
2022-03-24 16:45:28,673 - INFO -         factor : 1e-05
2022-03-24 16:45:28,673 - INFO -         function : L2
2022-03-24 16:45:28,673 - INFO - ------------------------------------------------------------
2022-03-24 16:45:28,673 - INFO - TRAIN : 
2022-03-24 16:45:28,673 - INFO -     batch_size : 1
2022-03-24 16:45:28,673 - INFO -     data_dir : ./dataset/flowers102/
2022-03-24 16:45:28,674 - INFO -     file_list : ./dataset/flowers102/train_list.txt
2022-03-24 16:45:28,674 - INFO -     num_workers : 4
2022-03-24 16:45:28,674 - INFO -     shuffle_seed : 0
2022-03-24 16:45:28,674 - INFO -     transforms : 
2022-03-24 16:45:28,674 - INFO -         DecodeImage : 
2022-03-24 16:45:28,674 - INFO -             channel_first : False
2022-03-24 16:45:28,674 - INFO -             to_np : False
2022-03-24 16:45:28,674 - INFO -             to_rgb : True
2022-03-24 16:45:28,675 - INFO -         RandCropImage : 
2022-03-24 16:45:28,675 - INFO -             size : 224
2022-03-24 16:45:28,675 - INFO -         RandFlipImage : 
2022-03-24 16:45:28,675 - INFO -             flip_code : 1
2022-03-24 16:45:28,675 - INFO -         NormalizeImage : 
2022-03-24 16:45:28,675 - INFO -             mean : [0.485, 0.456, 0.406]
2022-03-24 16:45:28,675 - INFO -             order : 
2022-03-24 16:45:28,676 - INFO -             scale : 1./255.
2022-03-24 16:45:28,676 - INFO -             std : [0.229, 0.224, 0.225]
2022-03-24 16:45:28,676 - INFO -         ToCHWImage : None
2022-03-24 16:45:28,676 - INFO - ------------------------------------------------------------
2022-03-24 16:45:28,676 - INFO - VALID : 
2022-03-24 16:45:28,676 - INFO -     batch_size : 1
2022-03-24 16:45:28,676 - INFO -     data_dir : ./dataset/flowers102/
2022-03-24 16:45:28,676 - INFO -     file_list : ./dataset/flowers102/val_list.txt
2022-03-24 16:45:28,677 - INFO -     num_workers : 4
2022-03-24 16:45:28,677 - INFO -     shuffle_seed : 0
2022-03-24 16:45:28,677 - INFO -     transforms : 
2022-03-24 16:45:28,677 - INFO -         DecodeImage : 
2022-03-24 16:45:28,677 - INFO -             channel_first : False
2022-03-24 16:45:28,677 - INFO -             to_np : False
2022-03-24 16:45:28,677 - INFO -             to_rgb : True
2022-03-24 16:45:28,677 - INFO -         ResizeImage : 
2022-03-24 16:45:28,678 - INFO -             resize_short : 256
2022-03-24 16:45:28,678 - INFO -         CropImage : 
2022-03-24 16:45:28,678 - INFO -             size : 224
2022-03-24 16:45:28,678 - INFO -         NormalizeImage : 
2022-03-24 16:45:28,678 - INFO -             mean : [0.485, 0.456, 0.406]
2022-03-24 16:45:28,678 - INFO -             order : 
2022-03-24 16:45:28,678 - INFO -             scale : 1.0/255.0
2022-03-24 16:45:28,679 - INFO -             std : [0.229, 0.224, 0.225]
2022-03-24 16:45:28,679 - INFO -         ToCHWImage : None
2022-03-24 16:45:28,679 - INFO - ------------------------------------------------------------
2022-03-24 16:45:28,679 - INFO - classes_num : 102
2022-03-24 16:45:28,679 - INFO - epochs : 20
2022-03-24 16:45:28,679 - INFO - image_shape : [3, 224, 224]
2022-03-24 16:45:28,679 - INFO - is_distributed : False
2022-03-24 16:45:28,679 - INFO - mode : train
2022-03-24 16:45:28,680 - INFO - model_save_dir : ./output/
2022-03-24 16:45:28,680 - INFO - pretrained_model : ./pretrained/VGG16_pretrained
2022-03-24 16:45:28,680 - INFO - save_interval : 5
2022-03-24 16:45:28,680 - INFO - topk : 5
2022-03-24 16:45:28,680 - INFO - total_images : 2040
2022-03-24 16:45:28,680 - INFO - use_cpu : False
2022-03-24 16:45:28,680 - INFO - use_gpu : False
2022-03-24 16:45:28,681 - INFO - use_xpu : True
2022-03-24 16:45:28,681 - INFO - valid_interval : 1
2022-03-24 16:45:28,681 - INFO - validate : True
/usr/local/lib64/python3.7/site-packages/paddle/fluid/framework.py:156: UserWarning: PaddlePaddle version 1.7.0 or higher is required, but 0.0.0 installed, Maybe you are using a develop version, please make sure the version is good with your code.
  (min_version, fluid_version.full_version))
[INFO][XPURT][cgpu_detected:66] dir '/baidu-cgpu/xpu0/0d35b4dc2e86569aa6f6d5568a2d73a0e80c22f5e02bf58c932ff524060585a2' not exists
W0324 16:45:28.942513   225 device_context.cc:187] Please NOTE: xpu device: 0
autotune_file{autotune_file.txt} set 438 records
autotune_file{autotune_file.txt} set 0 records
2022-03-24 16:45:31,120 - INFO - Loading parameters from ./pretrained/VGG16_pretrained...
2022-03-24 16:45:31,497 - WARNING - variable fc8_offset is already excluded automatically
2022-03-24 16:45:31,497 - WARNING - variable fc8_weights is already excluded automatically
2022-03-24 16:45:32,098 - INFO - Finish initing model from ['./pretrained/VGG16_pretrained']
import ujson error: No module named 'ujson' use json
autotune_file{autotune_file.txt} set 438 records
autotune_file{autotune_file.txt} set 0 records
[INFO][XPURT][cgpu_detected:66] dir '/baidu-cgpu/xpu1/0d35b4dc2e86569aa6f6d5568a2d73a0e80c22f5e02bf58c932ff524060585a2' not exists
autotune_file{autotune_file.txt} set 438 records
autotune_file{autotune_file.txt} set 0 records
[INFO][BKCL][/home/qa_work/workspace/xpu_build_dailyjob/bkcl_root/baidu/xpu/bkcl/src/net_socket.cpp:37] NET/Socket : ipv4 net card [enp15s0], ip[192.168.137.90]
[INFO][BKCL][/home/qa_work/workspace/xpu_build_dailyjob/bkcl_root/baidu/xpu/bkcl/src/net_socket.cpp:37] NET/Socket : ipv4 net card [virbr0], ip[192.168.122.1]
[INFO][BKCL][/home/qa_work/workspace/xpu_build_dailyjob/bkcl_root/baidu/xpu/bkcl/src/net_socket.cpp:37] NET/Socket : ipv4 net card [docker0], ip[172.17.0.1]
[INFO][BKCL][/home/qa_work/workspace/xpu_build_dailyjob/bkcl_root/baidu/xpu/bkcl/src/net_socket.cpp:44] NET/Socket : ipv6 net card [enp15s0], ip[fe80::ec91:a08d:61c:6508]
autotune_file{autotune_file.txt} set 438 records
autotune_file{autotune_file.txt} set 0 records
autotune_file{autotune_file.txt} set 438 records
autotune_file{autotune_file.txt} set 0 records
autotune_file{autotune_file.txt} set 438 records
autotune_file{autotune_file.txt} set 0 records
W0324 16:45:35.250291   225 build_strategy.cc:165] fusion_group is not enabled for Windows/MacOS now, and only effective when running with CUDA GPU.
W0324 16:45:35.250483   225 build_strategy.cc:424] fuse_bn_add_act_pass is only supported on GPU, skipped.
W0324 16:45:35.274974   225 fuse_all_reduce_op_pass.cc:76] Find all_reduce operators: 19. To make the speed faster, some all_reduce ops are fused during training, after fusion, the number of all_reduce ops is 4.
AUTOTUNE:(14155776, 888016, 1, 3, 224, 224, 64, 3, 3, 1, 1, 1, 1, 1, 1, 0) = wpinned_hf_bsp(4, 1, 256, 64)
Find Best Result in 146 choices, avg-conv-optime = 149 us
AUTOTUNE:(14155776, 888016, 1, 3, 224, 224, 64, 3, 3, 1, 1, 1, 1, 1, 1, 0) = wpinned_hf_bsp(4, 1, 192, 64)
Find Best Result in 146 choices, avg-conv-optime = 148 us
[WARN][XPURT][xpu_llwait:667] ioctl() fail, (715) Kernel execution timed out
[WARN][XPURT][xpu_lllaunch_async:551] ioctl() fail, (715) Kernel execution timed out
[WARN][XPURT][xpu_launch_async:359] fail on kernel ty=CDNN name='_Z22conv_int16_wpinned_bspIffffEvPKT_PKT0_PT1_iiiiiiiiPKfS9_PfS9_PKT2_S9_ii' ncl=4 nco=8
[WARN][XPURT][xpu_lllaunch_async:551] ioctl() fail, (715) Kernel execution timed out
[WARN][XPURT][xpu_launch_async:359] fail on kernel ty=CDNN name='_Z22conv_int16_wpinned_bspIffffEvPKT_PKT0_PT1_iiiiiiiiPKfS9_PfS9_PKT2_S9_ii' ncl=4 nco=8
[WARN][XPURT][xpu_llwait:667] ioctl() fail, (715) Kernel execution timed out

**Comment by adaxiadaxi** [comment]
@1chang37 同学你好，请问咱们还有vgg的训练问题嘛

**Comment by 1chang37** [comment]
有



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年04月11日 18:31 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [PaddlePaddle/PaddleClas] [kunlun ]  paddleclas 运行vgg16出错 (Issue #1785) |

@1chang37 同学你好，请问咱们还有vgg的训练问题嘛
